Update/onik_002.txt

```diff
@@ -675,15 +675,15 @@ void main()
 //　ここでは一切の隠し事はできないらしい＠
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "　ここでは一切の隠し事はできないらしい。",
-		   NULL, "Apparently there is absolutely nothing you can keep hidden here.", GetGlobalFlag(GLinemodeSp));
+		   NULL, "Apparently, there is absolutely nothing you can keep hidden here.", GetGlobalFlag(GLinemodeSp));
```

Comment:
Not sure about this. Technically I think the proper thing to do is add a comma, but a comma can be omitted when the writer wants to avoid simulating a pause between two clauses like this.